x-pack/elastic-agent/pkg/fleetapi/acker/fleet/fleet_acker.go

```diff
@@ -69,7 +78,14 @@ func (f *Acker) Ack(ctx context.Context, action fleetapi.Action) error {
 }
 
 // AckBatch acknowledges multiple actions at once.
-func (f *Acker) AckBatch(ctx context.Context, actions []fleetapi.Action) error {
+func (f *Acker) AckBatch(ctx context.Context, actions []fleetapi.Action) (err error) {
+	span, ctx := apm.StartSpan(ctx, "ackBatch", "app.internal")
+	defer func() {
+		if err != nil {
+			apm.CaptureError(ctx, err).Send()
+		}
```

Comment:
[Suggestion]
It's safe to call `apm.CaptureError` with a nil error and then call `Send`. Therefore it can be simplified.
```suggestion
		apm.CaptureError(ctx, err).Send()
```